php_phongo.c

```diff
@@ -837,7 +837,9 @@ void php_phongo_read_preference_prep_tagsets(zval *tagSets TSRMLS_DC) /* {{{ */
 		zval *tagSet;
 
 		ZEND_HASH_FOREACH_VAL(ht_data, tagSet) {
+			ZVAL_DEREF(tagSet);
```

Comment:
I opened [PHPC-870](https://jira.mongodb.org/browse/PHPC-870) to track future iteration changes.